source/manual/alerts/high-memory-for-application.html.md

```diff
@@ -0,0 +1,34 @@
+---
+owner_slack: "#2ndline"
+title: High memory for application
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2017-06-17
+review_in: 6 months
+---
+
+This alert triggers when the application process uses too much memory.
+
+## Fix memory leaks
+
+A common cause of the alerts are memory leaks. For Rails applications you
+can try diagnosing the error with [the derailed gem][derailed].
+
+[derailed]: https://github.com/schneems/derailed_benchmarks
+
+## Restart the application
+
+If you can't find the source of the memory leak you can restart the application.
```

Comment:
We could link to the fab scripts here for restarting apps

https://github.com/alphagov/fabric-scripts/blob/master/app.py#L13